hw/ip/kmac/data/kmac_testplan.hjson

```diff
@@ -184,6 +184,23 @@
       stage: V2
       tests: ["{name}_key_error"]
     }
+    {
+      name: sideload_invalid
+      desc: ''' Test kmac responses correctly sideloaded key is invalidated.
+            **Stimulus**:
+            - Configure kmac and send keymgr request.
+            - Wait until random FSM app state is entered.
+            - Invalidate the sideloaded key by forcing the .valid signal to 0.
+            - Repeat the above sequence a few times.
```

Comment:
In each of these iterations, a new random KMAC configuration is used and tested. Depending on the KMAC configuration, we are entering different FSM states which we want to test.

Reply:
That makes sense, maybe it would make sense to add something to make it more like "Repeat the above sequence a few times to try different configurations." ?

Reply:
Done, thanks for the feedback.

Reply:
I'm really sorry but I don't think it says anything about configurations yet?

Reply:
Should now be updated, I've just pushed the changes based on your feedback :-)